Source/BluetoothManager.swift

```diff
@@ -125,7 +125,7 @@ public class BluetoothManager {
      - parameter options: Optional scanning options.
      - returns: Infinite stream of scanned peripherals.
      */
-    public func scanForPeripherals(serviceUUIDs: [CBUUID]?, options: [String: AnyObject]? = nil)
+    public func scanForPeripherals(withServices serviceUUIDs: [CBUUID]?, options: [String: AnyObject]? = nil)
```

Comment:
The first one looks kind of odd 😆 
The second one is okay and I actually used that style in `Peripheral+Convenience.swift` (`public func monitorValueUpdateForCharacteristic(with identifier: CharacteristicIdentifier) -> Observable<Characteristic>`)

But I am not quite sure if it is expressive enough. (CBCentralManager also uses the current style: `func scanForPeripherals(withServices serviceUUIDs: [CBUUID]?, 
                options: [String : Any]? = nil)`)